lineman/app/services/ability_service.coffee

```diff
@@ -66,5 +66,9 @@ angular.module('loomioApp').factory 'AbilityService', (CurrentUser) ->
       (!membership.admin or membership.group().adminIds().length > 1) and
       (membership.user() == CurrentUser or @canAdministerGroup(membership.group()))
 
+    canDeactivateUser: ->
+     _.all CurrentUser.memberships(), (membership) ->
+       !membership.admin or membership.group().adminIds().length > 1
```

Comment:
aren't you going to need to serialize admin_ids or fetch admins for the group for this check to work? maybe we should serialize a property 'moreThanOneAdmin'